### PR TITLE
airflow-provider-vdk: VDKOperator execute method

### PR DESCRIPTION
**Pull request by gageorgiev**
This change implements the execute method for the VDKOperator.
This method triggers a job. execution and is ran when the operator
is used inside an Airflow DAG.

Testing done: tested locally, unit test

Signed-off-by: Gabriel Georgiev <gageorgiev@vmware.com>

### CODE REVIEW

**Review by mivanov1988 — APPROVED** [review]

**Comment by mivanov1988** [comment]
LGTM

File: projects/vdk-plugins/airflow-provider-vdk/vdk_provider/hooks/vdk.py [hunk 1]
```diff
@@ -44,7 +46,7 @@ def __init__(
         conn_id: str,
         job_name: str,
         team_name: str,
-        timeout: int = 1,  # TODO: Set reasonable default
+        timeout: int = 5,  # TODO: Set reasonable default
```
Comment (doks5) [inline projects/vdk-plugins/airflow-provider-vdk/vdk_provider/hooks/vdk.py:49]: Why the change?

File: projects/vdk-plugins/airflow-provider-vdk/tests/operators/test_vdkoperator.py [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+            job_name="test_job_name",
+            team_name="test_team_name",
+            asynchronous=False,
+            task_id="test_task_id",
+        )
+
+    @mock.patch(
+        "taurus_datajob_api.ApiClient.call_api", side_effect=call_api_return_func
+    )
+    def test_execute(self, mock_call_api):
```
Comment (doks5) [inline projects/vdk-plugins/airflow-provider-vdk/tests/operators/test_vdkoperator.py:70]: Maybe also test a failure case?

**Review by doks5 — COMMENTED** [review]
There is a failed test in the CI/C.

File: projects/vdk-plugins/airflow-provider-vdk/vdk_provider/hooks/vdk.py (hunk 1 above)
Comment (gageorgiev) [reply projects/vdk-plugins/airflow-provider-vdk/vdk_provider/hooks/vdk.py:49]: It seems the hook connection times out too easily, at least that's my impression from testing it.

**Comment by gageorgiev** [comment]
> There is a failed test in the CI/C.

Fixed it, it was caused by the VDKHook change to the start_job_execution method.
